src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -380,7 +380,7 @@ callExpressions:
 memberExpressions:
 [(7, 35), (7, 42)] [(7, 31), (7, 42)] arr.entries entries
 [(8, 10), (8, 13)] [(8, 2), (8, 13)] console.log log
-[(16, 34), (16, 46)] [(16, 19), (16, 47)] prefsBlueprint.accessorName accessorName
+[(16, 34), (16, 46)] [(16, 19), (16, 47)] prefsBlueprint.[accessorName] accessorName
```

Comment:
hmm, does this work?  `prefsBlueprint.[accessorName]`

Reply:
ummm nope ... it shouldn't ,hmmmm ... how did i missed this. :) will look into it. thanks!